### ISSUE TITLE
Build failure on master

### ISSUE THREAD

**Issue by sanmai-NL** [briansmith/ring#645]
```
running "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-static" "-I" "include" "-Wall" "-Wextra" "-std=c1x" "-Wbad-
function-cast" "-Wmissing-prototypes" "-Wnested-externs" "-Wstrict-prototypes" "-pedantic" "-pedantic-errors" "-Wall" "-Wextra" "-Wcast-alig
n" "-Wcast-qual" "-Wenum-compare" "-Wfloat-equal" "-Wformat=2" "-Winline" "-Winvalid-pch" "-Wmissing-declarations" "-Wmissing-field-initiali
zers" "-Wmissing-include-dirs" "-Wredundant-decls" "-Wshadow" "-Wsign-compare" "-Wundef" "-Wuninitialized" "-Wwrite-strings" "-fno-strict-al
iasing" "-fvisibility=hidden" "-Wno-cast-align" "-fstack-protector" "-g3" "-Werror" "-D_XOPEN_SOURCE=700" "-c" "-o/Rust/target/debug/build/ring-d507c3d2e8f1b089/out/gfp_p384.o" "crypto/fipsmodule/ec/gfp_p384.c"

--- stderr
third_party/fiat/curve25519.c:442:13: error: 'fe_mul_llt' defined but not used [-Werror=unused-function]
 static void fe_mul_llt(fe_loose *h, const fe_loose *f, const fe *g) {
             ^~~~~~~~~~
cc1: all warnings being treated as errors
```

**Comment by briansmith** [comment]
This should be fixed as of 6faa06b72774e7e444ba845846ee55331dbe1266. See also bd38b9e41992013ac39be254264bede44881c893.

**Comment by briansmith** [comment]
Thanks for reporting this!
